tracer/src/Datadog.AutoInstrumentation.NativeLoader/loader.conf

```
@@ -3,6 +3,6 @@ PROFILER;{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A};win-x64;.\ContinuousProfiler\Dat
 PROFILER;{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A};win-x86;.\ContinuousProfiler\Datadog.AutoInstrumentation.Profiler.Native.x86.dll
 
 #Tracer
-TRACER;{846F5F1C-F9AE-4B07-969E-05C26BC060D8};win-x64;.\tracer\win-x64\Datadog.Trace.ClrProfiler.Native.dll
-TRACER;{846F5F1C-F9AE-4B07-969E-05C26BC060D8};win-x86;.\tracer\win-x86\Datadog.Trace.ClrProfiler.Native.dll
+TRACER;{50DA5EED-F1ED-B00B-1055-5AFE55A1ADE5};win-x64;.\tracer\win-x64\Datadog.Trace.ClrProfiler.Native.dll
+TRACER;{50DA5EED-F1ED-B00B-1055-5AFE55A1ADE5};win-x86;.\tracer\win-x86\Datadog.Trace.ClrProfiler.Native.dll
```

Tracer with capital "T"? Seems like a common theme throughout this PR 😅 
```suggestion
TRACER;{50DA5EED-F1ED-B00B-1055-5AFE55A1ADE5};win-x64;.\Tracer\win-x64\Datadog.Trace.ClrProfiler.Native.dll
TRACER;{50DA5EED-F1ED-B00B-1055-5AFE55A1ADE5};win-x86;.\Tracer\win-x86\Datadog.Trace.ClrProfiler.Native.dll
```

---

Yep good catch!